{{cookiecutter.repo_name}}/.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+data_file = .coverage
+omit = {{cookiecutter.repo_name}}/settings*
+    {{cookiecutter.repo_name}}/conf*
+    *wsgi.py
+    *migrations*
+    *admin.py
+    *test*
```

Comment:
Two benefits: finding tests that aren't run, and measuring the test helpers that often live in the tests directories.